tests/default/_core/update_by_query.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
```

Comment:
This is _update_by_query/rethrottle, so it should be in `tests/default/_core/_update_by_query/rethrottle.yaml`.